base-editors/nano/autobuild/beyond

```diff
@@ -0,0 +1,5 @@
+abinfo "Installing nanorc..."
+mkdir -pv "$PKGDIR"/etc
+mkdir -pv "$PKGDIR"/usr/share/nano
+install -vm644 "$SRCDIR"/../nanorc/* "$PKGDIR"/usr/share/nano
```

Comment:
You could have merged these commands with, for example...

```
install -Dvm644 "$SRCDIR"/../nanorc/*.nanorc \
    -t "$PKGDIR"/usr/share/nano/
```

Also, this line also installs `nanorc` to /usr/share/nano, which is incorrect.